src/App.jsx

```diff
@@ -22,14 +24,21 @@ const extractedAlbums = albums.map((album) => {
     coverImage,
   };
 
-});
+});*/
 
-console.log(extractedAlbums);
+//console.log(extractedAlbums);
 
+//-------------------Extracted albums finished?--------------------//
 
-//header to export? Make one?
+//What shows in the browser?
 export const App = () => {
-  return <div>
-    Hallo!
-  </div>;
+  const albums = data.albums.items
```

Comment:
I had this code here. I think you should return HTML rather than to have a new function within the function (although I do not understand 100% myself either ) export const App = () => {
  return (
    <div>
      <Album data={data.albums.items} />
    </div>)
}